cpp/tests/utilities/type_lists.hpp

```diff
@@ -87,6 +90,40 @@ auto make_type_param_vector(std::initializer_list<T> const& init_list)
   return vec;
 }
 
+template <typename TypeParam, typename T>
+typename std::enable_if<cudf::is_timestamp_t<TypeParam>::value, std::vector<TypeParam>>::type
+make_type_param_vector(std::initializer_list<T> const& init_list)
+{
+  std::vector<TypeParam> vec(init_list.size());
+  std::transform(std::cbegin(init_list), std::cend(init_list), std::begin(vec), [](auto const& e) {
+    return TypeParam{typename TypeParam::duration{e}};
+  });
+  return vec;
+}
+
+/**
+ * @brief Convert the numeric value of type T to a fixed width type of type TypeParam.
+ *
+ * @param init_value Value used to initialize the fixed width type
+ * @return A fixed width type - [u]int32/float/duration etc. of type TypeParam with the
+ *         value specified
+ */
+template <typename TypeParam, typename T>
+typename std::enable_if<cudf::is_fixed_width<TypeParam>() &&
+                          !cudf::is_timestamp_t<TypeParam>::value,
+                        TypeParam>::type
+make_type_param_scalar(T const init_value)
+{
+  return static_cast<TypeParam>(init_value);
+}
+
+template <typename TypeParam, typename T>
+typename std::enable_if<cudf::is_timestamp_t<TypeParam>::value, TypeParam>::type
```

Comment:
Where is `make_type_param_scalar` necessary?

Reply:
this is used in a number of typed tests that tries to construct a fixed width type with one/more integer values. those are changed now to use this method with the appropriate `TypeParam` type.